webapp/views/App/views/Data/Charts/components/Panel/components/RawChartBuilder/RawChartBuilder.js

```diff
@@ -127,7 +127,7 @@ const RawChartBuilder = ({ spec, onUpdateSpec, dimensions }) => {
   }, [helpOpened])
 
   return (
-    <div className="raw-chart-builder">
+    <div className={`raw-chart-builder ${visible ? 'visiable' : ''}`}>
```

Comment:
there is a typo here: "visiable". Why don't you use classNames for this? it would become something like: 
className={classNames( 'raw-chart-builder', { visible } ) }